analyticConfigurations/ace-efs.json

```diff
@@ -0,0 +1,60 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "aws.efs.burstcreditbalance.*",
```

Comment:
Do you need the `.*` at the end? `aws.efs.burstcreditbalance` is the full metric FQN.